snuba/datasets/entities/discover.py

```diff
@@ -314,6 +312,7 @@ def __init__(self) -> None:
                 # Other tags and context
                 ("tags", Nested([("key", String()), ("value", String())])),
                 ("contexts", Nested([("key", String()), ("value", String())])),
+                ("trace_id", String(Modifiers(nullable=True))),
```

Comment:
This is the common columns list used for for Discover entity selection. `trace_id` was missing from here earlier (we weren't really using it before since we couldn't format the UUID properly). But it is actually on both errors and transactions and is formatted correctly now so if we get a query for it we can use the Discover storage, and this line ensures we are selecting it.

Reply:
I see. OK and it's a string type because it's not promoted everywhere?

Reply:
I made it a string because that's it's external type. UUID is an internal only implementation detail. Though I don't think this information in the entity schema is ever actually used anywhere.